src/http/api/views/account.posts.view.ts

```diff
@@ -604,7 +613,7 @@ export class AccountPostsView {
                 url: result.author_url ?? '',
                 avatarUrl: result.author_avatar_url ?? '',
             },
-            authoredByMe: result.author_id === accountId,
+            authoredByMe: result.author_id === contextAccountId,
```

Comment:
Another way of solving this could be to use the `post_reposted_by_current_user` / `post_liked_by_current_user` pattern, currently in use for `repostedByMe` / `likedByMe` - that way we don't need to pass any account ID to `mapToPostDTO` and that logic stay in the query?